### ISSUE TITLE
Quick question about Camera and Gallery mode

### ISSUE THREAD

**Issue by voghDev** [Drjacky/ImagePicker#65]
Hello!

first of all, thanks for this awesome library, it's doing a great task.
I just want to ask a question about the documentation in the README file.
The example code provided is the following :point_down: 
```
        pickImageLauncher.launch(
            ImagePicker.with(requireActivity())
                .createIntent()
        )
```
it states: **If you want both Camera and Gallery**

But in my case, the code  **directly uses the camera** instead of **allowing me to choose between camera and Gallery**. It is working well though.
:question: :grey_question:  Is this the intended behaviour? :question: :grey_question: 

Tested devices were Xiaomi Mi A3 (Android 11), Xiaomi Mi A2 Lite (Android 10) and Nexus 5X emulator (Api 26).

Additionally, I see that `.compress()` [was removed](https://github.com/Drjacky/ImagePicker/commit/b784e8ff2c2205cc1f6266616af824c2b78065e5#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR10). Is there a replacement for it? I'm using `maxResultSize` but am not sure if it's the best option

Thank you so much!

**Comment by voghDev** [comment]
A few more information and context in case it helps.

This code worked well allowing us to select a picture from Gallery in the mentioned devices:
```
    pickImageLauncher.launch(
            ImagePicker.with(requireActivity())
                .maxResultSize(
                    1024,
                    1024,
                    false
                )
                .cropSquare()
                .galleryOnly()
                .galleryMimeTypes(mimeTypes = someMimeTypes)
                .createIntent()
        )
```
where `pickImageLauncher` is:
```
private val pickImageLauncher =
        registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
            if (result.resultCode == Activity.RESULT_OK) {
               // ...
           }
        }
```

**Comment by Drjacky** [comment]
Where do you see this:
`pickImageLauncher.launch(`
?

What I see in the readme.md is:
```kotlin
If you want both Camera and Gallery:

Kotlin

    ImagePicker.with(this)
       //...
       .createIntentFromDialog { launcher.launch(it) }
```

**Comment by voghDev** [comment]
:heavy_check_mark:  I finally solved my issue. I'll tell you how. Something really curious was happening :scream: :point_down: 

I tried the code on the first comment because the sample code wasn't working for me. When I launched it, nothing was working. A dialog to choose between Camera and Gallery was supposed to appear, and it wasn't appearing :point_down: 

```
ImagePicker.with(requireActivity())
            .maxResultSize(
                1024,
                1024,
                false
            )
            .cropSquare()
            .galleryMimeTypes(mimeTypes = someMimeTypes)
            .createIntentFromDialog { pickImageLauncher.launch(it) }
```
After introspecting in the library code and setting some breakpoints, I found a curious value for `imageProvider` in `ImagePicker::createIntentFromDialog` function:

![provider](https://user-images.githubusercontent.com/4963568/163203782-c3fb4deb-696d-414b-a0bd-8970ba05ea20.jpg)

I can't guess why :sweat_smile: because I don't see `FRONT_CAMERA` assigned to `imageProvider` anywhere, but the Intent extras.
Anyway, all I needed to do was to force `imageProvider` to have `BOTH` as value. Code is as follows:

```
ImagePicker.with(requireActivity())
            .maxResultSize(
                1024,
                1024,
                false
            )
            .provider(ImageProvider.BOTH) // This line is the key
            .cropSquare()
            .galleryMimeTypes(mimeTypes = someMimeTypes)
            .createIntentFromDialog { pickImageLauncher.launch(it) }
```
Now the dialog is showing and everything works again :slightly_smiling_face: Thank you very much

**Comment by Drjacky** [comment]
It has been fixed on [2.1.16](https://github.com/Drjacky/ImagePicker/releases/tag/2.1.16)
